README.md

```diff
@@ -8,13 +8,16 @@ Drone plugin for FOSSA.
 
 The following snippet should be applied to the `.drone.yml` file:
 
+#### NOTE: This will run `fossa analyze` on every drone-pr AND drone-publish run
```

Comment:
We shouldn't mention drone-pr and drone-publish since this is a generic drone plugin for anyone wishing to use FOSSA.